src/Apps/Home/Components/HomeNewWorksFromGalleriesYouFollowRail.tsx

```diff
@@ -0,0 +1,156 @@
+import { Shelf, Skeleton } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "System/useSystemContext"
+import { useTracking } from "react-tracking"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { HomeNewWorksFromGalleriesYouFollowRail_newWorksFromGalleriesYouFollowConnection$data } from "__generated__/HomeNewWorksFromGalleriesYouFollowRail_newWorksFromGalleriesYouFollowConnection.graphql"
+import { HomeNewWorksFromGalleriesYouFollowRailQuery } from "__generated__/HomeNewWorksFromGalleriesYouFollowRailQuery.graphql"
+import {
+  ShelfArtworkFragmentContainer,
+  ShelfArtworkPlaceholder,
+} from "Components/Artwork/ShelfArtwork"
+import { extractNodes } from "Utils/extractNodes"
+import {
+  ActionType,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+import { Rail } from "Components/Rail/Rail"
+
+interface HomeNewWorksFromGalleriesYouFollowRailProps {
+  newWorksFromGalleriesYouFollowConnection: HomeNewWorksFromGalleriesYouFollowRail_newWorksFromGalleriesYouFollowConnection$data
+}
+
+const HomeNewWorksFromGalleriesYouFollowRail: React.FC<HomeNewWorksFromGalleriesYouFollowRailProps> = ({
+  newWorksFromGalleriesYouFollowConnection,
+}) => {
+  const { trackEvent } = useTracking()
+
+  const artworks = extractNodes(newWorksFromGalleriesYouFollowConnection)
+
+  if (!artworks || artworks?.length === 0) {
+    return null
+  }
+
+  return (
+    <Rail
+      title="New Works From Galleries You Follow"
+      viewAllLabel="View All Works"
+      viewAllHref="/new-works-from-galleries-you-follow"
+      viewAllOnClick={() => {
+        const trackingEvent: ClickedArtworkGroup = {
+          action: ActionType.clickedArtworkGroup,
+          context_module: ContextModule.newWorksByGalleriesYouFollowRail,
+          context_page_owner_type: OwnerType.home,
+          destination_page_owner_type: OwnerType.collection,
+          destination_page_owner_id: "932d0b13-3cf1-46d1-8e49-18b186230347",
+          destination_page_owner_slug: "curators-picks-emerging",
```

Comment:
This doesn't match the slug it actually points to?

Reply:
Thanks, I've fixed the tracking.